odoo/http.py

```diff
@@ -703,7 +709,7 @@ def dispatch(self):
                 else:
                     rpc_request.debug(logline)
 
-            return self._json_response(result)
+            return self._json_response(result, duration=end_time-start_time)
```

Comment:
Please add spaces before and after the `-` operation, otherwise it's hard to read.